src/Matterhorn/Events.hs

```diff
@@ -155,8 +156,8 @@ formatError (NoSuchHelpTopic topic) =
     let knownTopics = ("  - " <>) <$> helpTopicName <$> helpTopics
     in "Unknown help topic: `" <> topic <> "`. " <>
        (T.unlines $ "Available topics are:" : knownTopics)
-formatError (BadFile file) =
-    "Specified file " <> file <> " either doesn't exist or couldn't be read"
+formatError (BadAttachmentPath e msg) =
+    msg <> " The exception encountered was:\n " <> T.pack (displayException e)
```

Comment:
Would it be possible to make this message inspect the exception? These functions come to mind:

https://hackage.haskell.org/package/base-4.14.1.0/docs/System-IO-Error.html#g:2

I'm thinking that the raw `Show` of the exception will probably not be what the user wants to see (and may not know how to act on it), but if we can translate common exceptions into more user-friendly language, that would be nice. (An easy one is "file not found")

Reply:
I've done a small refactor to store only the exception instead of the message.  Now the messages are generated by querying the exception.